plugins/docker/docker.plugin.zsh

```diff
@@ -0,0 +1,42 @@
+alias di='docker info'
+alias dlg='docker container logs'
+alias dls='docker container ls'
+alias dlsa='docker container ls -a'
+alias dr='docker container run'
+alias drit='docker container run -it'
+alias drm='docker container rm'
+alias drmf='docker container rm -f'
+alias dst='docker container start'
+alias dstp='docker container stop'
+alias dt='docker top'
+alias dv='docker version'
+alias dpo='docker container port'
+alias dpu='docker pull'
+alias dx='docker container exec'
+alias dxit='docker container exec -it'
+alias dbl='docker build'
+alias dhh='docker help'
+alias dcin='docker container inspect'
+alias dpsa='docker container ps -a'
```

Comment:
Just a question here: what's the difference with `docker ps -a`?

Reply:
It's the same thing. The new docker commands format is `docker <command> <subcommand>` , so I've tried to use that in all the aliases. 

However, your comment got me thinking that we don't need ps because we already have aliases for ls and ls -a. So I've removed it.